admin/views.py

```diff
@@ -75,7 +77,9 @@ def attest_invoice_part(request, pk):
     try:
         invoice_part = InvoicePart.objects.get(pk=int(pk))
     except ObjectDoesNotExist:
-        raise Http404("Fakturadelen finns inte")
```

Comment:
You are basically replicating the default behavior of django here, but also removing the debug functionality on errors.

How it works is if `DEBUG=False` (as it is in production) when this gets called:
```python
raise Http404("Fakuradelen finns inte")
```
django would run this function: 
```python
defaults.page_not_found(request, exception, template_name='404.html')
```
which calls the template `404.html` with the variables `request_path` and `exception` (in this case `"Fakturadelen finns inte"`)

so it would render something like this with your updated template:
![cashflow-403-2](https://user-images.githubusercontent.com/23638900/141193801-444732df-b37d-4a61-97ec-858aa8ea75fe.png)

Similar things work for other errors but have different variables available by default (can be overridden):
- 400.html (nothing passed to template by default)
- 403.html (`exception` passed to template)
- 500.html (nothing passed to template by default)

Here are some more resources about it: [Error Views](https://docs.djangoproject.com/en/3.2/ref/views/#error-views).

But as it is written it would remove stack traces when you have `DEBUG=True` in development.

Reply:
Worth mentioning is that it looks like when a 404 is triggered by not being able to resolve the path, it still has an exception but it is called `Resolver404`. So this is how that would look:
![cashflow-403-3](https://user-images.githubusercontent.com/23638900/141196220-dc02f40e-2e5c-4752-bdad-bdaffac5eb41.png)